doc/conf.py

```diff
@@ -0,0 +1,223 @@
+# Configuration file for the Sphinx documentation app.
+
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+from datetime import datetime
+import inspect
+import os
+from os.path import relpath, dirname
+import re
+import sys
+
+from pyxem import release_info
+import pyxem
+import requests
+
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+sys.path.append("../")
+
+
+# Project information
+project = "pyxem"
+copyright = release_info.copyright
+author = release_info.author
+version = release_info.version
+release = release_info.version
+
+master_doc = "index"
+
+# Add any Sphinx extension module names here, as strings. They can be extensions
+# coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
+extensions = [
+    "sphinxcontrib.bibtex",
+    "sphinx.ext.autodoc",
+    "sphinx.ext.autosummary",
+    "sphinx.ext.intersphinx",
+    "sphinx.ext.mathjax",
+    "sphinx.ext.napoleon",
+    "sphinx.ext.linkcode",
+    "sphinx_autodoc_typehints",
+    "sphinx_copybutton",
+    "sphinx_gallery.load_style",
+    "nbsphinx",
+    'sphinx.ext.viewcode',
```

Comment:
Either use this extension, which link to source code like [Matplotlib does it](https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.arrow.html#matplotlib.pyplot.arrow) (press the "[source]" link), on the web page, or like [SciPy does it](https://docs.scipy.org/doc/scipy/reference/generated/scipy.ndimage.labeled_comprehension.html#scipy-ndimage-labeled-comprehension), linking to the source code on GitHub. The latter uses the `linkcode_resolve()` function below. We use the latter in kikuchipy.